src/app/containers/NewStoryPromo/PromoOrganisms/TopStoriesPromo/TopStoriesItem.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { shape, number, string } from 'prop-types';
+import pathOr from 'ramda/src/pathOr';
+import { storyItem } from '#models/propTypes/storyItem';
+import { buildUniquePromoId } from '#app/containers/StoryPromo/utilities';
+import { getIsLive } from '#lib/utilities/getStoryPromoInfo';
+import Promo from '../../Promo';
+import { StyledPromoHeading } from './index.styles';
+
+const TopStoriesItem = ({ item, index, labelId }) => {
+  const { script, translations } = useContext(ServiceContext);
+
+  const overtypedHeadline = pathOr('', ['headlines', 'overtyped'], item);
+  const headline =
+    overtypedHeadline || pathOr('', ['headlines', 'headline'], item);
+
+  const mediaType = pathOr(null, ['media', 'format'], item);
+  const mediaDuration = pathOr(null, ['media', 'duration'], item);
+  const isPhotoGallery = pathOr(null, ['cpsType'], item) === 'PGL';
+
+  const timestamp = pathOr(null, ['timestamp'], item);
+  const url = pathOr(null, ['locators', 'assetUri'], item);
+  const isLive = getIsLive(item);
+
+  const liveLabel = pathOr('LIVE', ['media', 'liveLabel'], translations);
+
+  // As screenreaders mispronounce the word 'LIVE', we use visually hidden
+  // text to read 'Live' instead, which screenreaders pronounce correctly.
+  const liveLabelIsEnglish = liveLabel === 'LIVE';
+
+  const linkId = buildUniquePromoId({
+    sectionType: 'top-stories',
+    promoGroupId: labelId,
+    promoItem: item,
+    promoIndex: index,
+  });
+
+  return (
+    <Promo to={url} id={linkId}>
+      <Promo.ContentWrapper>
+        {mediaType && <Promo.MediaIndicator type={mediaType} />}
+        <StyledPromoHeading script={script}>
+          <Promo.Link>
+            {isLive ? (
+              <Promo.LiveLabel
+                liveText={liveLabel}
+                ariaHidden={liveLabelIsEnglish}
+                offScreenText={liveLabelIsEnglish ? 'Live' : null}
+              >
+                <Promo.Content
+                  mediaType={mediaType}
+                  mediaDuration={mediaDuration}
+                  headline={headline}
+                  isPhotoGallery={isPhotoGallery}
+                />
```

Comment:
Rather than passing props here could content accept children here and the TopStories specific logic be moved up to this level? If you look at `RelatedContentPromo` you can see very [few props are passed](https://github.com/bbc/simorgh/pull/10165/files#diff-3f534fe85ed6a29519ed3b50fa4b4e8864b48beb45ede29cc935e995df82c8e0R110-R117) which implies these props are specific to TopStoriesPromo use case. 

We want to push 'use case' specific logic as high as possible and leave the lower level components clean with reusable logic applicable everywhere. 

We are seeing a bit of Open/Closed principle violation here I think.